src/library_path.js

```diff
@@ -64,10 +64,9 @@ addToLibrary({
       return root + dir;
     },
     basename: (path) => {
-      // EMSCRIPTEN return '/'' for '/', not an empty string
+      // EMSCRIPTEN return '/' for '/', not an empty string
```

Comment:
BTW this is another inconsistency with the NODERAWFS version.. makes me wonder if we can remove this too?   Unrelated to this change though.

Reply:
Well, coreutils basename returns `/` for `/` and `//`  too. Our code returns '/' for input `/` but empty string for input //. If we wanted to be really careful about it we'd need something like:
```js
if (path === '') return '';
path = path.replace(/\/+$/g, "");
if (path === '') return '/';
```

Reply:
Lets leave that as a possible followup.